elements/bernardi-guzman-neilan-raugel.def

```diff
@@ -0,0 +1,50 @@
+name: Bernardi-Guzman-Neilan-Raugel
```

Comment:
@pbrubeck What do you think of this name for the "Guzman-Neilan" element currently in Symfem? Once we agree on a name, I'll update the name used in Symfem itself to match

Reply:
that's a better suggestion. The element implemented in FIAT is not "Guzman-Neilan (first kind)" with alternative name "Guzman-Neilan". The element currently implemented almost correctly in Symfem is "Guzman-Neilan (second kind)"

Reply:
The element in FIAT is Guzman-Neilan extended to (Pk + FacetBubble)^d, the first (dim+1)^2 dofs give you GN